src/mpeInstrument/index.js

```diff
@@ -3,6 +3,8 @@ import { generateMidiActions } from './actions';
 import { logger } from './middlewares';
 import rootReducer from './reducers';
 
+const ALL_NOTES_OFF = new Uint8Array([0xb0, 123, 0]);
```

Comment:
It may be nice to move this and all of the midi message consts from the functional tests into one place.